docs/indepth/installation.rst

```diff
@@ -379,6 +379,58 @@ Your firewall will need to allow access to the ports your applications are expos
    Keep in mind the consequences of exposing unsecured services to the Internet.
    Both applications with exposed ports and applications accessed via links will be accessible by anyone on the Internet.
 
+ZFS Backend Configuration
+-------------------------
+
+The ZFS backend requires ZFS to be installed.
+
+
+Installing ZFS on CentOS 7
+..........................
+
+Installing ZFS requires the kernel development headers for the running kernel.
+Since CentOS doesn't provide easy access to old package versions,
+the easiest way to get appropriate headers is to upgrade the kernel and install the headers.
+
+.. task:: upgrade_kernel centos-7
+   :prompt: [root@centos-7]#
+
+You will need to reboot the node after updating the kernel.
+
+.. prompt:: bash [root@centos-7]#
+
+   shutdown -r now
+
+You must also install the ZFS package repository.
+
+.. task:: install_zfs centos-7
+   :prompt: [root@centos-7]#
+
+
+Installing ZFS on Ubuntu 14.04
+..............................
+
+.. task:: install_zfs ubuntu-14.04
+   :prompt: [root@ubuntu-14.04]#
+
+
+Creating a ZFS Pool
+...................
+
+Flocker requires a ZFS pool.
+The pool is typically named named ``flocker`` but this is not required.
+The following commands will create a 10 gigabyte ZFS pool backed by a file:
+
+.. task:: create_flocker_pool_file
```

Comment:
:question: It might be clearer what can be replaced in these commands if `task_create_flocker_pool_file` were changed to take a variable pool name, and these instructions created a variable and then set them.